rfc/README.md

```diff
@@ -117,4 +117,5 @@ The list of all RFCs can be found here.
 | 79         | [Robust handling of spark task retries and failures](./rfc-79/rfc-79.md)                                                                                                                                             | `IN PROGRESS`  |
 | 80         | [Column Families](./rfc-80/rfc-80.md)                                                                                                                                                                                | `UNDER REVIEW` |
 | 81         | [Log Compaction with Merge Sort](./rfc-81/rfc-81.md)                                                                                                                                                                 | `UNDER REVIEW` |
-| 82         | [Concurrent schema evolution detection](./rfc-82/rfc-82.md)                                                                                                                                                                                | `UNDER REVIEW` |
+| 82         | [Concurrent schema evolution detection](./rfc-82/rfc-82.md)                                                                                                                                                          | `UNDER REVIEW` |
+| 83         | [Hudi data is ordered by primary key](./rfc-83/rfc-83.md)                                                                                                                                                            | `UNDER REVIEW` |
```

Comment:
There is already a RFC-83 for completion time based incremental queries, should we reuse the RFC-81 slot though?

Reply:
> There is already a RFC-83 for completion time based incremental queries, should we reuse the RFC-81 slot though?

Got it, thanks, I will close the PR now